yb-voyager/src/srcdb/ora2pg_export_data.go

```diff
@@ -63,6 +63,14 @@ func ora2pgExportDataOffline(ctx context.Context, source *Source, exportDir stri
 	defer utils.WaitGroup.Done()
 
 	configFilePath := filepath.Join(exportDir, "temp", ".ora2pg.conf")
+	customConfigFilePath := filepath.Join(exportDir, "metainfo", "base-ora2pg.conf")
+	if utils.FileOrFolderExists(customConfigFilePath) {
+		utils.PrintAndLog("Using custom base-ora2pg.conf for data migration.")
+		source.PopulateOra2pgConfigFile(customConfigFilePath)
+		configFilePath = customConfigFilePath
+	} else {
+		source.PopulateOra2pgConfigFile(configFilePath)
+	}
 	source.PopulateOra2pgConfigFile(configFilePath)
```

Comment:
`PopulateOra2pgConfigFile()` is getting called twice?

Reply:
This was not intentional, I don't think my changes reflected due to some internet issues I was having at the time of pushing the relevant patch